mathesar/api/exceptions/exceptions.py

```diff
@@ -24,18 +29,25 @@ def get_default_exception_detail(exception, error_code=ErrorCodes.NonClassifiedE
 
 
 def get_default_api_exception(exc):
-    return MathesarAPIException(exc, ErrorCodes.NonClassifiedError.value)
+    return MathesarAPIException(exc, ErrorCodes.UnknownError.value)
 
 
 class MathesarAPIException(APIException):
```

Comment:
Since this is the base class, I think it would make sense to move this to a `base.py` file, editing this exception will affect everything else.

Reply:
I refactored exceptions.py and moved the expectations into appropriate modules